tests/models/modules/test_SwaVFreezing.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+
+import torch
+
+from lightly.models.modules.heads import SwaVPrototypes
+
+class TestSwavFreezingPrototypes(unittest.TestCase):
+    def test_swav_frozen_prototypes(self, device: str = "cpu", seed=0):
```

Comment:
Could you move the test also to `test_ProjectionHeads`? Tests for functions from the same file should also always be in the same file.